chopper_tune.cfg

```diff
@@ -50,6 +50,11 @@ gcode:
     {% else %}
         {% set stepper = 'stepper_' + params.STEPPER|string|lower %} ; Import default stepper for search its driver
     {% endif %}
+
+    {% if axis == 'e'%}
```

Comment:
It is not entirely clear to me why you did not connect this block with the block of code above, using elif

Reply:
Well, okay, you can do that. But I think we'll have to rework this block when I integrate delta kinematics into the script properly